tests/integration/test_sizing.py

```diff
@@ -213,13 +213,13 @@ def _check_sizes(self, block_index, expectations, expected_img_width=None, is_de
                     container_width=item_bank_width,
                     expected_percent=expect.width_percent,
                 )
-            if expect.fixed_width_percent is not None:
-                self._check_width(
-                    item_description="Unplaced item {} with fixed width".format(expect.item_id),
-                    item=self._get_unplaced_item_by_value(expect.item_id),
-                    container_width=target_img_width,
```

Comment:
@bradenmacdonald i was fixing the tests when I noticed this line in the section where the unplaced items are being tested. Shouldnt the unplaced items be tested against `item_bank_width` rather than `target_img_width`?

Reply:
@cgopalan I'm not sure, but that sounds like it would make more sense, yep.

Reply:
@cgopalan I think `target_img_width` is correct, since the percent is supposed to be relative to the background image, see https://github.com/open-craft/xblock-drag-and-drop-v2/blob/671ff9bd8bbbf7c53cc3c8977eb586f77a6f7a82/drag_and_drop_v2/public/js/drag_and_drop.js#L34-L40.